src/Security/PersonTokenAuthenticator.php

```diff
@@ -144,4 +118,43 @@ public function onAuthenticationFailure(Request $request, AuthenticationExceptio
             403
         );
     }
+
+    /**
+     * Authentication success.
+     *
+     * @param Request        $request     A Symfony Request, req by interface.
+     * @param TokenInterface $token       A Symfony user token, req by interface.
+     * @param string         $providerKey The name of the used firewall key.
+     *
+     * @return Response The response or null to continue request.
+     */
+    // Next line to be ignored because implemented function does not have type-hint on $providerKey.
+    // phpcs:ignore
+    public function onAuthenticationSuccess(Request $request, TokenInterface $token, $providerKey)
+    {
+        return;
```

Comment:
" If you return null, the current request will continue, and the user
will be authenticated. This makes sense, for example, with an API" 

Above comment is from the AuthenticatorInterface, so i think we can return a null response?

Reply:
Yes, it can be null. There is not additional checking needed. I the token is valid, it's done. But this function HAS to be implemented, so something has to be returned.

Reply:
From the documentation,
`    public function onAuthenticationSuccess(Request $request, TokenInterface $token, $providerKey): ?Response
    {
        // on success, let the request continue
        return null;
    }`

I think it is more elegant returning null and it would match the return type as well

Reply:
In an example there was no return statement at all, and it had this same comment. Can you make a code suggestion?

Reply:
I got that example code from this 
https://symfony.com/doc/4.4/security/guard_authentication.html

Reply:
I'm unable to make multi-line code suggestion, it only lets me do it for single line